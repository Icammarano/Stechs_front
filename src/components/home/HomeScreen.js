import React from "react";
import logo from "../../images/stechs.png";
import "../../styles/components/Home.css";

export const HomeScreen = () => {
    return (
        <div>
            <div className="container home__main">
                <h1>Stechs Vendors</h1>
                <hr />
                <img src={logo} alt="Logo" className="home__image" />
            </div>
        </div>
    );
};
