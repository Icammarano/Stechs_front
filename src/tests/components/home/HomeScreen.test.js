import { shallow } from "enzyme";
import { HomeScreen } from "../../../components/home/HomeScreen";

describe("Pruebas en <HomeScreen />", () => {
    test("Debe mostrarse correctamente", () => {
        const wrapper = shallow(<HomeScreen />);
        expect(wrapper).toMatchSnapshot();
    });
});
