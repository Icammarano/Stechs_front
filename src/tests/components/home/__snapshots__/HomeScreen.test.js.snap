// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <HomeScreen /> Debe mostrarse correctamente 1`] = `
<div>
  <div
    className="container home__main"
  >
    <h1>
      Stechs Vendors
    </h1>
    <hr />
    <img
      alt="Logo"
      className="home__image"
      src="stechs.png"
    />
  </div>
</div>
`;
